recipes/sarge/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "sarge" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "59f93216723ddd9062d17cbbb90ed9e69267b84825cf0bde0b7f8d934c424823" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: pip install --no-deps .
```

Comment:
Is there a reason to prefer that over pip? AIUI using pip is always recommended now to install packages. This is also what we do [for the ipython feedstock](https://github.com/conda-forge/ipython-feedstock/blob/8121ecd5043b96fd7fe9f075a56ef09eaf016995/recipe/meta.yaml#L14), among a number of other recipes.